src/tioga.C

```diff
@@ -212,7 +212,7 @@ void tioga::performConnectivityAMR(void)
   MPI_Barrier(scomm);
   //printf("Finished performConnectivityAMR in %d\n",myid);
   //ierr=0;
-  MPI_Abort(scomm,ierr);
+//  MPI_Abort(scomm,ierr);
```

Comment:
How is this related to the qnode/qcell change?

Reply:
@sayerhs Thanks for catching this! This should have been a separate pull request. I have reverted that commit.